docs/guides/rest_authc_authz.md

```diff
@@ -0,0 +1,69 @@
+# rest_authc_authz
+
+## Description
+
+The rest authenticator rely on an external rest endpoint to validate user credentials.
```

Comment:
s/rely/relies/ and I would complete "to validate user credentials and retrieves user groups"